pravega/src/test/scala/docs/scaladsl/PravegaGraphSpec.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.scaladsl
+
+import akka.stream.alpakka.pravega.{ReaderSettingsBuilder, WriterSettingsBuilder}
+import akka.stream.alpakka.pravega.scaladsl.Pravega
+import akka.stream.scaladsl.{Keep, Sink, Source}
+import io.pravega.client.stream.impl.JavaSerializer
+import akka.stream.KillSwitches
+
+import scala.concurrent.{Await, Promise}
+import scala.concurrent.duration.DurationInt
+
+class PravegaGraphSpec extends PravegaBaseSpec {
+
+  val serializer = new JavaSerializer[String]
+
+  // #configuration
+
+  implicit val readerSettings = ReaderSettingsBuilder(sys, b => ())
+    .withSerializer(serializer)
+
+  implicit val writerSettings = WriterSettingsBuilder(sys)
+    .withSerializer(serializer)
+
+  val writerSettingsWithRoutingKey = WriterSettingsBuilder(sys)
+    .withSerializer(serializer, (str: String) => str.substring(0, 2))
+
+  // #configuration
+
+  "Pravega connector" should {
+
+    "runs sources" in {
+
+      logger.info("start source")
+
+      // #writing
+      Source(1 to 100).map(i => f"$i%02d_event").runWith(Pravega.sink(scope, streamName))
+
+      Source(1 to 100)
+        .map(i => f"$i%02d_event")
+        .runWith(Pravega.sink(scope, streamName)(writerSettingsWithRoutingKey))
```

Comment:
```suggestion
      Source(0 to 99).map(i => f"$i%02d_event").runWith(Pravega.sink(scope, streamName))

      Source(0 to 99)
        .map(i => f"$i%02d_event")
        .runWith(Pravega.sink(scope, streamName)(writerSettingsWithRoutingKey))
```

`f"$i%02d"` only formats 2 digits and the key extractor uses the first 2 characters.

Reply:
Formatting was removed.